src/js/index.js

```diff
@@ -1,7 +1,16 @@
+import loadBreeds from './modules/load-breeds-home';
+import '../style.css';
+import pagination from './modules/pagination-home';
+
 const addItem = () => {
   const newItem = '<li>Mientras miro las nuevas olas</li>';
   const list = document.getElementById('list');
   list.innerHTML += newItem;
 };
```

Comment:
Please, remove the stub code from the project setup

Reply:
Ok :+1: